### ISSUE TITLE
Export a separate component without prefixing.

### ISSUE THREAD

**Issue by Aubron** [planttheidea/react-style-tag#15]
Currently the style component includes an option for not prefixing code. This is great! Prefixing may not be required in all circumstances, and the bloat to the bundle for on-the-fly prefixing is significant.

I am proposing a separate component without prefixing, as even with the prop set to disable prefixing, using the component adds an untenable amount of code to generated bundles. Even if you are not using prefixing in your application, including the `<Style />` tag will add the entirety of caniuse-db to your bundle, as it is imported regardless.

**Comment by Aubron** [comment]
As an aside, if you're not using browser prefixes, https://github.com/nfl/react-helmet is a pretty good alternative.

**Comment by planttheidea** [comment]
@Aubron this similarly should be alleviated when the rewrite is complete to use `stylis`, which has a much smaller footprint. Hopefuly that should be sufficient, because what u ask will be basically impossible with the new setup ... `stylis` serves as both prefixer (optionally), minifier, and normalizer (you can use `&` for sass-like nesting of styles), we can only be as efficient as stylis is.

And to your aside, there are a TON of great alternatives out there. I'm only aware of `stylis` because that is was `styled-components` uses under the hood, which is now my standard for scoped styles.

**Comment by planttheidea** [comment]
@Aubron I have just published `2.0.0` which includes the aforementioned bundle improvements, however I did not incorporate an "unprefixed style" component as requested because (a) it would not impact bundle size, but also (b) this can be accomplished with a simple thin abstraction within your application:

```javascript
import React from 'react';
import {Style} from 'react-style-tag';

export const UnprefixedStyle = (props) => {
  return (
    <Style
      {...props}
      isPrefixed={false}
   />
  );
};
```

I'll leave this issue open until you let me know one way or another if this resolves the issue.

**Comment by Aubron** [comment]
Sounds good, thanks!
